testing/ibus/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Bart Ribbers <bribbers@disroot.org>
+pkgname=ibus
+pkgver=1.5.18
+pkgrel=0
+pkgdesc="Next Generation Input Bus for Linux"
+arch="all"
+url="https://github.com/ibus/ibus/wiki"
+license="LGPL-2.1"
+depends_dev="gnome-common gtk+2.0-dev gtk+3.0-dev dconf-dev libnotify-dev iso-codes-dev gobject-introspection-dev"
+makedepends="$depends_dev autoconf automake intltool gtk-doc libtool python3 unicode-emoji cldr-emoji-annotation unicode-character-database gconf coreutils dbus-x11 vala glib-dev "
+source="$pkgname-$pkgver.tar.gz::https://github.com/$pkgname/$pkgname/archive/$pkgver.tar.gz
+	ucd-path.patch::https://github.com/ibus/ibus/commit/5788be80685f397c3db3bdf4e672d67cfb9b3433.patch"
+subpackages="lib$pkgname $pkgname-dev $pkgname-doc $pkgname-lang"
+options="!check"
+
+prepare() {
+	default_prepare
+
+	cd "$builddir"
+	sed -i 's|$(libibus) $(libibus_emoji_dialog)|$(libibus_emoji_dialog) $(libibus)|' ui/gtk3/Makefile.am
```

Comment:
Can we patch the Makefile.am instead for sed'ing it? So we notice if this is fixed upstream (whatever it is supposed to fix).